jobs/sync-integration-tests/templates/config.json.erb

```diff
@@ -2,12 +2,8 @@
   require 'json'
   config = p('sync_integration_tests.config')
 
-  bbs_client_cert_contents = "/tmp/bbs_client_cert"
-  bbs_client_key_contents = "/tmp/bbs_client_key"
-  config[:bbs_client_cert] = bbs_client_cert_contents
-  config[:bbs_client_key] = bbs_client_key_contents
-  File.write(bbs_client_cert_contents, config.fetch("bbs_client_cert_contents"))
-  File.write(bbs_client_key_contents, config.fetch("bbs_client_key_contents"))
+  config[:bbs_client_cert] = "/var/vcap/jobs/sync-integration-tests/bin/bbs_client_cert"
+  config[:bbs_client_key] = "/var/vcap/jobs/sync-integration-tests/bin/bbs_client_key"
```

Comment:
Manual rubocop warning: use `'` instead of `"` for strings with no interpolation :)